pkg/virtctl/guestfs/guestfs.go

```diff
@@ -45,47 +45,46 @@ const (
 	tmpDirVolumeName  = "libguestfs-tmp-dir"
 	tmpDirPath        = "/tmp/guestfs"
 	pullPolicyDefault = corev1.PullIfNotPresent
+	timeout           = 500 * time.Second
 )
 
-var (
+type guestfsCommand struct {
+	clientConfig clientcmd.ClientConfig
+	podParams    libguestfsPodParams
+}
+
+type libguestfsPodParams struct {
```

Comment:
Not sure about this level of nesting. IMO it is fine to keep these values in the `guestfsCommand` struct.

Reply:
but then need to send lot of parameters to the function, i dont think it is good idea